### DISCUSSION TITLE
Solution for some issues while using your phone directly to test app instead of simulator

### DISCUSSION THREAD

**Discussion by ty-zhao1** [CS342/.github#31] [Technical Questions] [answered]
### In what area do you have a technical challenge?

Xcode, Simulator & Previews

### Description

Assuming you have set up a free developer account and linked it to Xcode, 

If you’re running into this error:

Automatic signing failed. Xcode failed to provision this target. Please file a bug report at <https://feedbackassistant.apple.com> and include the Update Signing report from the Report navigator,

Here’s the steps you can take:

1. Navigate to your application target.
2. Go to the Signing and Capabilities Page
3. Go to the debug tab
4. Click the trash can next to HealthKit

Now Xcode should build and run to your connected device, and you can add HealthKit back afterwards by clicking on + Capability (next to the debug button).

If, upon download, there is a further notification that says something along the lines of “unable to verify” or can’t trust the app, follow these steps:
1. Go into settings onto the device, search up VPN and Device Management. 
2. Indicate that you can trust the incoming app/developer (I don’t remember exactly what it says but there’s a button that says trust this device)
3. Click verify and now you should be able to run the app no issues.
4. If this still doesn’t work, make sure all software is up to date, then restart both the phone and Xcode, and if that doesn’t work restart the computer.

Hope this helps!

### Reproduction

If you just try to run the simulator to a device plugged into your computer, this problem should occur.

### Expected behavior

One expects the app to just load on the phone and be ready, but it runs out there are some extra steps necessary as outlined above.

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment, marked as answer]
Thank you @ty-zhao1 👍
